scripts/setup/check_environment.py

```
@@ -37,12 +39,12 @@ def check_dependencies():
 
 def check_env_file():
     """检查 .env 文件"""
-    env_file = Path(__file__).parent.parent / ".env"
+    env_file = PROJECT_ROOT / ".env"
     if env_file.exists():
         print(f"✅ .env 文件存在")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)